src/bindings/python/tests/test_inference_engine/test_tensor.py

```diff
@@ -268,3 +269,68 @@ def test_cannot_set_shape_incorrect_dims():
     with pytest.raises(RuntimeError) as e:
         ov_tensor.shape = [3, 28, 28]
     assert "Dims and format are inconsistent" in str(e.value)
+
+
+@pytest.mark.parametrize("ov_type", [
+    (ov.Type.u1),
+    (ov.Type.u4),
+    (ov.Type.i4),
+])
+def test_cannot_create_roi_from_packed_tensor(ov_type):
+    ov_tensor = Tensor(ov_type, [1, 3, 48, 48])
+    with pytest.raises(RuntimeError) as e:
+        Tensor(ov_tensor, [0, 0, 24, 24], [1, 3, 48, 48])
+    assert "ROI Tensor for types with bitwidths less then 8 bit is not implemented" in str(e.value)
+
+
+@pytest.mark.parametrize("ov_type", [
+    (ov.Type.u1),
+    (ov.Type.u4),
+    (ov.Type.i4),
+])
+def test_cannot_get_strides_for_packed_tensor(ov_type):
+    ov_tensor = Tensor(ov_type, [1, 3, 48, 48])
+    with pytest.raises(RuntimeError) as e:
+        ov_tensor.get_strides()
+    assert "Could not get strides for types with bitwidths less then 8 bit." in str(e.value)
+
+
+@pytest.mark.parametrize("dtype", [
+    (np.uint8),
+    (np.int8),
+    (np.uint16),
+    (np.uint32),
+    (np.uint64),
+])
+@pytest.mark.parametrize("ov_type", [
+    (ov.Type.u1),
+    (ov.Type.u4),
+    (ov.Type.i4),
+])
+def test_init_with_packed_buffer(dtype, ov_type):
+    shape = [1, 3, 32, 32]
+    fit = np.dtype(dtype).itemsize * 8 / ov_type.bitwidth
+    assert np.prod(shape) % fit == 0
+    size = int(np.prod(shape) // fit)
+    buffer = np.random.normal(size=size).astype(dtype)
+    ov_tensor = Tensor(buffer, shape, ov_type)
+    assert ov_tensor.data.nbytes == ov_tensor.byte_size
+    assert np.array_equal(ov_tensor.data.view(dtype), buffer)
+
+
+@pytest.mark.parametrize("shape", [
+    ([1, 3, 28, 28]),
+    ([1, 3, 27, 27]),
+])
+@pytest.mark.parametrize("low, high, ov_type, dtype", [
+    (0, 2, ov.Type.u1, np.uint8),
+    (0, 16, ov.Type.u4, np.uint8),
+    (-8, 7, ov.Type.i4, np.int8),
+])
+def test_packing(shape, low, high, ov_type, dtype):
+    ov_tensor = Tensor(ov_type, shape)
+    data = np.random.uniform(low, high, shape).astype(dtype)
+    packed_data = pack_data(data, ov_tensor.element_type)
+    ov_tensor.data[:] = packed_data
+    unpacked = unpack_data(ov_tensor.data, ov_tensor.element_type, ov_tensor.shape)
```

Comment:
Are you able to add a paragraph to Python's exclusives section about u1/u4/i4 data types and how to handle them?